win32/readdir.c

```diff
@@ -127,46 +127,6 @@ struct dirent *readdir(DIR *dp)
 	return &(dp->dent);
 }/*}}}*/
 
-int readdir_r(DIR *dp, struct dirent *entry, struct dirent **result)
```

Comment:
It doesn't look right to rid `readdir_r`, since it calls `php_win32_cp_conv_w_to_any` (which is important for the long and UTF-8 path support). Instead the function may be aliased to `readdir` (or such). AFAIK, MSVC doesn't support `readdir` anyway. /cc @weltling

Reply:
There is a `readdir` function directly above this one, which also calls php_win32_cp_conv_w_to_any, which will be used now. Or am I misunderstanding something here?

Reply:
Oops, missed that. Should be okay then. Thanks!